03-make_epochs.py

```diff
@@ -42,11 +42,10 @@ def run_epochs(subject, session=None):
                                            run=run,
                                            processing=config.proc,
                                            recording=config.rec,
-                                           space=config.space
-                                           )
+                                           space=config.space)
         # Prepare a name to save the data
-        raw_fname_in = \
-            op.join(deriv_path, bids_basename + '_filt_raw.fif')
+        raw_fname_in = op.join(deriv_path,
+                               bids_basename.update(suffix='filt_raw.fif'))
```

Comment:
you can use prefix for `deriv_path` I guess?

Reply:
But it's not a prefix, it's the BIDS root…

Reply:
Not sure what you mean but see: https://github.com/mne-tools/mne-bids/issues/454. I think `prefix` should do the job for you. Whether it's an appropriate name is another question.

Reply:
Done in a057d87